submissions/karelskiy/HTML and CSS practice/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
+    <script defer src="script.js"></script>
+    <title>Document</title>
+</head>
+
+<body>
+    <a href="#"><img id="Doe" width="35px" height='35px'
+            title="John Doe" src="img/JhonDoe.jpg"
+            alt=""></a>
```

Comment:
Empty `alt` attribute of `img`  tag is bad practice. Fill all `alt` attributes of `img` tags.